integration-tests/src/bin/benchmark/main.rs

```diff
@@ -65,6 +65,8 @@ fn main() {
         })
         .collect();
 
+    std::thread::sleep(std::time::Duration::from_millis(250));
```

Comment:
is this leftover after debuging?

Reply:
It's a hack that makes the segfaults less prevalent. We figured that some of the segfaults were caused by threads not being awaited for on drop